modules/indexing/src/test/java/org/apache/ignite/internal/metric/SqlStatisticsMemoryQuotaTest.java

```diff
@@ -300,6 +302,14 @@ private long longMetricValue(int gridIdx, String metricName) {
         return ((LongMetric)metric).value();
     }
 
+    private long maxMemory(int gridIdx) {
+        return ((IgniteH2Indexing) grid(gridIdx).context().query().getIndexing()).memoryManager().memoryLimit();
```

Comment:
Before your change these tests check metrics like the user do (see longMetricValue method above)
Now, this test do wrong checks. IgniteH2Indexing is internal API and user never do this.
I bet memory metrics is not available via MetricsAPI as SqlMemoryStatisticsHolder doesn't registers them anymore.